### ISSUE TITLE
 Establish authoritative sources for each variant (or at least US and GB)

### ISSUE THREAD

**Issue by jaumeortola** [languagetool-org/english-pos-dict#9]

**Comment by jaumeortola** [comment]
This table is useful to understand the main differences between variants: https://en.wikipedia.org/wiki/Oxford_spelling#Language_tag_comparison

Hunspell GB dictionary includes both Oxford and British spellings. 

More lists of differences to check: 
https://blog.collinsdictionary.com/language-lovers/9-spelling-differences-between-british-and-american-english/
https://www.examenglishforfree.com/the-differences-between-british-english-and-american-english-spelling/

**Comment by AzadehSafakish** [comment]
Might be useful:
Here is the full us2uk list I put together for [this issue](https://github.com/languagetooler-gmbh/languagetool-premium/issues/7098).
I think as a general rule: Anything that's a US spelling can be applied to CA (unless the entry specifically says otherwise), anything that's a GB spelling can applied to AU, NZ, ZA (unless the entry specifically says otherwise).
[us2uk_expanded.json](https://github.com/languagetool-org/english-pos-dict/files/15321363/us2uk_expanded.json)

**Comment by jaumeortola** [comment]
> Anything that's a US spelling can be applied to CA
 
Is this general enough? According to the Wikipedia table, en-CA matches en-US in `organization, realize, aging, analyze`, but not in `defence, licence, traveller`.

**Comment by AzadehSafakish** [comment]
>Is this general enough?

No, you're right, it's too general. Canadian English is a bit more of a mash-up.
Wikipedia is the only "nice" table I can find at the moment; other sources are far less concise.

[Editing Canadian English](https://www.amazon.co.uk/Editing-Canadian-English-3rd-Everyone-ebook/dp/B07665VS7H/ref=sr_1_3?crid=Q35KVKYQTVLY&dib=eyJ2IjoiMSJ9.5dJk4uDwC0X8OB0Zfv3mO6seXtVze043V4LTAOzcXBeEwOQywolR3kqVrKQOnpOwliJF2Zqg1kIkBeEeTgCoCx_DY41sjIhRmOyV7sPLPbm189H2SypfSW50k4P1n6czeFyNcA0gY2U60Tq6qR3eERggm9qcROsbnalkK3-uMfWmrajxcjCuEBhI7hflKiKBGvqUW7AQ1ol64ea91uf4zf9xCaGUZgnCKMqFNWAJaHg.cPpUlFYKM7TtGo4X0ZPCmOV4tcaPiH9ZjNq_qwf1xp0&dib_tag=se&keywords=canadian+english&qid=1716281617&s=books&sprefix=canadian+englis%2Cstripbooks%2C244&sr=1-3) (Read sample > 3/4 of the way to the bottom: '3.5 Variant spellings by category') also has a few tables, but they more or less seem to agree with Wikipedia.

The Wikipedia table might be the simplest and quickest solution for CA, unless anyone can find an easy online resource for the Canadian Oxford Dictionary (that we can get a word list from).
